nats-streaming/README-short.txt

```diff
@@ -1 +1,2 @@
+DEPRECATED;
 NATS Streaming is an open-source, high-performance, cloud native messaging streaming system.
```

Comment:
Sorry, Docker Hub has a limit of 100 characters; how about this?

```suggestion
DEPRECATED; NATS Streaming is a high-performance, cloud native messaging streaming system.
```

Reply:
Sorry, just saw that in your README, so removed "NATS Streaming" from the sentence, since it is redundant because it is already in the NATS Streaming page. Should be good now.